### DISCUSSION TITLE
Moving content from one instance to another

### DISCUSSION THREAD

**Discussion by shodson** [OrchardCMS/OrchardCore#9195] [Q&A]
What is the best way to move content from one instance of OrchardCore to another instance on another server? I don't want a full copy and replace, rather, I want to copy some select content and put it on to the other instance, without having to resort to a bunch of copy and paste keyboard/mouse work. 

BTW, I've tried manipulating the SQL. I copied the JSON docs from the ContentItemIndex and Document tables from the source server, copied them into my tables on the other server, made sure the document IDs in both tables matched each other, and set the ListContentItemId to the ID of it's parent in the dest server, but they won't show up in the CMS admin on the destination server.

FYI, these instances are still running RC1, which begs my next question, what is the best way to migrate to RC2? I've been waiting for several months for a v1.0 release but that seems to be nowhere in sight so I'll guess I'll have to settle for RC2 for now.

**Comment by Skrypt** [comment]
Have you restarted the Application Pool after executing the SQL Query? It could be a server cache issue.

**Reply by shodson** [reply]
Yes, it's an App Service in Azure, I "Reset" it, that should have done it.

**Reply by Skrypt** [reply]
The ListPart maybe has changed it's JSON structure ; take a look at this. I think that might be a good leading trail.

**Comment by hishamco** [comment]
What about using Deployment Plan?

**Reply by shodson** [reply]
Wouldn't that just be a complete copy of all content of the source, copying over all of the already-existing content on the destination? If so, that won't work, I want to select certain pieces of content to copy over. Is there a way to do that with deployment plans?

**Comment by deanmarcussen** [comment]
I would suggest upgrading to RC2.

Going from RC1 direct to a V1 isn't going to be a recommended option.

There are many improvements to deployment plans in RC2, including adding single content items to plans, deploy directly from the admin list, and many more.

Technically the thing you are probably missing in your manual steps, is to update the `ContainedPartIndex` as well.
